template/.helix/languages.toml

```diff
@@ -0,0 +1,7 @@
+#INCLUDEFILE helix
+[[language]]
+name = "rust"
+
+[language-server.rust-analyzer.config]
+check.allTargets = false
+cargo.target = "riscv32imac-unknown-none-elf"
```

Comment:
```suggestion
check.allTargets = false
#REPLACE riscv32imac-unknown-none-elf rust_target
cargo.target = "riscv32imac-unknown-none-elf"
```

Reply:
Right, I saw that in the vscode one but for some reason I forgot to think about what it did